tests/libunwind/host/host.cpp

```diff
@@ -106,7 +106,11 @@ int main(int argc, const char* argv[])
     {
         oe_put_err("oe_create_libunwind_enclave(): result=%u", result);
     }
+#ifdef _WIN32
```

Comment:
I would like to not have any ifdef here.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/posix-getpid?view=vs-2019 explains:
"The Microsoft-implemented POSIX function name getpid is a deprecated alias for the _getpid function. By default, it generates Compiler warning (level 3) C4996. The name is deprecated because it doesn't follow the Standard C rules for implementation-specific names. However, the function is still supported."

So one approach is to just violate posix and use getpid in both windows and Linux.
Another approach would be to follow posix guidelines and use _getpid in both cases, and if some platform (like Linux) violates posix guidelines then that platform can have a macro in a .h file that redefines _getpid to their posix-violating name (getpid).
Another approach would be to define oe_getpid (since there is no standard) and map it in a .h file to different names on different platforms.
But in any case the point is to define one common name in a .h file and make .c and .cpp files avoid platform specific ifdefs.